lib/mana/engines/javascript.rb

```
@@ -21,38 +67,210 @@ def self.create_context
+          })
+          attached << "ruby.__ref_alive"
+        end
+
+        unless attached.include?("ruby.__ref_to_s")
+          ctx.attach("ruby.__ref_to_s", proc { |ref_id|
+            obj = registry.get(ref_id)
+            obj ? obj.to_s : "released"
+          })
+          attached << "ruby.__ref_to_s"
+        end
+      end
```

RuboCop (1.84.2)

[convention] Assignment Branch Condition size for `attach_ref_callbacks` is too high. [<11, 27, 7> 29.98/23]

(Metrics/AbcSize)